tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+minversion = 3.7.0
+skip_missing_interpreters = true
+envlist =
+    py{37,38,39}
+
+[testenv]
+passenv = DD_SOURCE_*  DD_DESTINATION_*
+setenv =
+    DD_HTTP_CLIENT_RETRY_TIMEOUT = 300
+    MAX_WORKERS = 1
+    RECORD = {env:RECORD:false}
+usedevelop = true
+deps =
+    click==7.1.2
+    click-config-file==0.6.0
+    requests==2.25.1
+    deepdiff==5.5.0
+    black>=20.8b1
+    pytest>=5.2.4
+    pytest-black
+    pytest-console-scripts
+    pytest-recording
+    pytest-env
```

Comment:
can you update extra dependencies in `setup.cfg`?

Reply:
nothing to update there apparently